apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -1,25 +1,129 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { useWallet } from '@mysten/wallet-adapter-react';
+import { WalletWrapper } from '@mysten/wallet-adapter-react-ui';
+import clsx from 'clsx';
+import { useMemo } from 'react';
+import { useFieldArray } from 'react-hook-form';
+import toast from 'react-hot-toast';
+import { z } from 'zod';
+
+import { FunctionParamInput } from './FuncitonParamInput';
+import { useFunctionParamsDetails } from './useFunctionParamsDetails';
+
 import type { SuiMoveNormalizedFunction, ObjectId } from '@mysten/sui.js';
 
+import { useZodForm } from '~/hooks/useZodForm';
+import { Button } from '~/ui/Button';
 import { DisclosureBox } from '~/ui/DisclosureBox';
 
+const argsSchema = z.object({
+    params: z.array(z.object({ value: z.string().trim().min(1) })),
```

Comment:
Shouldn't min be 0? There are param-less functions.

Reply:
min 1 refers to the value of the param, 0 params work fine. With this there is no way though to have optional params (not sure if necessary though).